melos.yaml

```diff
@@ -13,6 +13,13 @@ command:
     # branch: master
     # Additionally build a changelog at the root of the workspace.
     workspaceChangelog: true
+
+    changelogs:
+      - path: firebase_ui_changelog.md
+        scope: firebase_ui_*
```

Comment:
This isn't valid anymore btw, scope should now be nested under a packageFilters map, e.g.: 


![image](https://user-images.githubusercontent.com/5347038/205098351-a0b530f6-e72d-4798-95a6-0e1738f72403.png)